### ISSUE TITLE
Turn on/off light only works if you don't specify "light" or "lights" or if the entity_id ends with "lights"

### ISSUE THREAD

**Issue by Aephir** [home-assistant/intents#864]
I have a light group, `light.dining_room`, and another `light.kitchen_lights`.

If I say `Turn off kitchen lights`, it works fine. If I try with `Turn off dining room lights`, I get an error:
<img width="395" alt="Screenshot 2023-02-02 at 10 14 10" src="https://user-images.githubusercontent.com/20152788/216282636-c8360b24-33ff-44ac-8409-d349967c7a20.png">
But if I say `Turn off dining room `, it works fine. I'm just worried it would turn off other stuff than lights, e.g., switches.

From what I can see [here](https://github.com/home-assistant/intents/blob/main/sentences/en/light_HassTurnOn.yaml), `Turn on the dining room lights` should work?

I've tested this with several lights. It so far only works if the entity_id of the light ends with `lights`, despite the domain being `light` for all.

**Comment by synesthesiam** [comment]
Is your light entity named "Dining Room"?

**Comment by Aephir** [comment]
@synesthesiam yes, it has entity_id: `light.dining_room` with friendly_name: `Dining Room`.

I thought the fact that they are in domain = light would mean you could/should say `turn on XX lights`, regardless of whether they had the word `light` entity_id/friendly_name to differentiate other "things" in that room that might all turn on/off if not specifying domain in command?

**Comment by nagyrobi** [comment]
Yes, I've ran into this myself too.

**Comment by schizza** [comment]
As no language have sentence for turning on one light entity, it won't work. The intents will fall to HassTurnOn I suppose.
This is what needs to be fixed. 
in light_TurnOn there should be sentences for single entity with {name}. There are only areas.

**Comment by schizza** [comment]
This is #770 issue you are talking about, I think

**Comment by Aephir** [comment]
Yes, I think you might be right.

**Comment by TheFes** [comment]
I have solved this for Dutch by optionally adding all variations to refer to lights after the name.
I will try to find some time to add this to English as well
